packages/auth/src/api/index.test.ts

```diff
@@ -346,10 +346,10 @@ describe('api/_performApiRequest', () => {
       clock.restore();
     });
 
-    it('should handle network failure', async () => {
+    it('should handle failures', async () => {
```

Comment:
Maybe call this "should handle non-FirebaseErrors" and move it to line 308? 

This need not be since the "context('with network issues',"

Reply:
Done.